docs/accessibility/accessibility-tools.md

```diff
@@ -58,7 +58,10 @@ Google's Chrome and Microsoft's Edge browsers have several accessibility tools b
 
 <figure>
   <uxdot-example width-adjustment="900px">
-    <img src="/assets/accessibility/lighthouse-audit.png" alt="Screenshot of a Lighthouse autdit with a score of 100%.">
+    <img src="/assets/accessibility/lighthouse-audit.png"
+      alt="Screenshot of a Lighthouse autdit with a score of 100%."
+      width="900"
+      height="251">
```

Comment:
Make sure to have these attributes properly aligned ala [wiki guidelines](https://github.com/RedHat-UX/red-hat-design-system/wiki/HTML-Formatting#attributes).

That goes for this `img` tag and the rest of the images in this PR.

Reply:
Side note: does this alignment need to be added to the 1st PR in this series?

Reply:
From the guidelines:
> Attributes should be indented by at least two units.

What are you seeing different?

Reply:
> Columnar Attributes
Attributes _may_ line up with the first inline attribute, so long as they are indented by as least two units. This is a matter of taste which can produce more beautiful and easier to understand templates.

In a recent PR, I aligned image attrs to look like they do here and Benny aligned them in a vertically straight column in a subsequent commit.